flags/eigendaflags/cli.go

```diff
@@ -163,10 +181,96 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			EnvVars:  []string{withEnvPrefix(envPrefix, "PUT_RETRIES")},
 			Category: category,
 		},
+	},
+		v2Flags(envPrefix, category)...,
+	)
+}
+
+func v2Flags(envPrefix, category string) []cli.Flag {
+	return []cli.Flag{
+		// EigenDA V2 specific flags //
+		&cli.BoolFlag{
+			Name:     V2Enabled,
+			Usage:    "Enable blob dispersal and retrieval against EigenDA v2 protocol",
```

Comment:
couldn't one want to allow v2 retrievals but still want to disperse to v1? feels like we should have 2 flags here?

Reply:
eh why would we wanna allow someone to migrate between V1 <--> V2 if we intend to nuke V1?

Reply:
Circling back here after last week's discussion. When testing migrations we said we'd want to support a canary option which keeps sending some % of certs to V1. cc @litt3

Reply:
@samlaf Are you on board with implementing this as part of a future PR?

Reply:
yes

Reply:
https://linear.app/eigenlabs/issue/DAINT-324/implement-v1-v2-canary-testing

Reply:
Why not created on github itself? :(